homeassistant/const.py

```diff
@@ -273,6 +273,7 @@ class Platform(StrEnum):
 CONF_WEEKDAY: Final = "weekday"
 CONF_WHILE: Final = "while"
 CONF_WHITELIST: Final = "whitelist"
+CONF_ZIP_CODE: Final = "zip_code"
```

Comment:
Please move this constant to a const.py module in the justnimbus integration. Two few integrations are using it for it to be made common.